utils_cv/detection/model.py

```diff
@@ -803,7 +803,7 @@ def load(self, name: str = None, path: str = None) -> None:
             self.labels = meta_data["labels"]
 
     @classmethod
-    def from_saved_model(cls, name: str, path: str) -> "DetectionLearner":
+    def from_saved_model(cls, name: str, path: str, mask: bool = False) -> "DetectionLearner":
```

Comment:
Ideally the file "meta.json" could contain if this is a mask model or not. I guess this PR is time-critical hence signing-off now.

Reply:
yes, time critical, thanks, we can look into enhancements later.